Tangem/App/ViewModels/WalletModel/WalletModel+ExpressWallet.swift

```diff
@@ -0,0 +1,69 @@
+//
+//  WalletModel+ExpressWallet.swift
+//  Tangem
+//
+//  Created by Sergey Balashov on 10.11.2023.
+//  Copyright © 2023 Tangem AG. All rights reserved.
+//
+
+import Foundation
+import TangemSwapping
+import BlockchainSdk
+
+extension TokenItem {
+    var expressCurrency: TangemSwapping.ExpressCurrency {
+        switch self {
+        case .blockchain(let blockchain):
+            return TangemSwapping.ExpressCurrency(
+                // Fixed constant value for the main token contract address
+                contractAddress: ExpressConstants.coinContractAddress,
+                network: blockchain.networkId
+            )
+        case .token(let token, let blockchain):
+            return TangemSwapping.ExpressCurrency(
+                contractAddress: token.contractAddress,
+                network: blockchain.networkId
+            )
+        }
+    }
+}
+
+extension WalletModel: ExpressWallet {
+    var currency: TangemSwapping.ExpressCurrency {
```

Comment:
думаю тут лучше быть более явным
```suggestion
    var expressCurrency: TangemSwapping.ExpressCurrency {
```

Reply:
✅